.appveyor.yml

```diff
@@ -68,7 +68,7 @@ install:
           Add-Content php.ini "`n extension=php_curl.dll"
           Add-Content php.ini "`n curl.cainfo=C:\tools\cacert\bundle.pem"
 
-          $DLLVersion = "5.5.0preview"
+          $DLLVersion = "5.6.1"
```

Comment:
Please revert unless this is required. If possible, we should roll back to using the latest stable version as it was prior to #3484 but that would be a different PR.

Reply:
@morozov I reverted, but probably some tests won't pass.

Reply:
@morozv Now in appveyor is the error:
`
PHP Fatal error:  Meta data vector out of sync in
`
Should I create a new pull request with a change of version?